### DISCUSSION TITLE
Bundle scripts wrong loading order [Pimcore update 10 to 11.1.5]

### DISCUSSION THREAD

**Discussion by valentin-** [pimcore/pimcore#16617] [Q&A / Support]
Hi, I'm getting a script error in the backend from the script-proxy content as customreports is used before it got loaded. 

Line 226:
pimcore.bundle.googlemarketing.report.analytics.elementexplorer = Class.create(pimcore.bundle.customreports.abstract, {

Line 10000:
pimcore.registerNS("pimcore.bundle.customreports.abstract");

It worked earlier but something messed it up and I can't find it. I've tried to reorder the Bundles in bundles.php, add them to Kernel.php with priority. Uninstall/reinstall bundles, removing the bundles folder and reinstall, removing the admin scripts.

Anyone got an idea what could cause the issue?

Thanks!

**Comment by kingjia90** [comment]
Could it be fixed by https://github.com/pimcore/pimcore/pull/16546?

**Reply by valentin-** [reply]
Thanks for your reply. I installed the dev branch with the changes and tried to reorder the bundles, but PimcoreGoogleMarketingBundle alway got loaded first in the script - no matter how i changed the order.

At the end I uninstalled/installed some bundles. It looks like uninstalling the GoogleMarketingBundle solved my issue.

console pimcore:bundle:uninstall PimcoreGoogleMarketingBundle
